ingestion/functions/aws.py

```diff
@@ -11,7 +11,10 @@
 import common.common_lib as common_lib
 
 AWS_REGION = "us-east-1"
-AWS_IMAGE = "612888738066.dkr.ecr.us-east-1.amazonaws.com/gdh-ingestor:latest"
+AWS_IMAGE = {
+    'dev': "612888738066.dkr.ecr.us-east-1.amazonaws.com/gdh-ingestor:latest",
+    'prod': "612888738066.dkr.ecr.us-east-1.amazonaws.com/gdh-ingestor:stable"
+}
```

Comment:
Does this change update the already-deployed batch jobs, or is there a manual intervention needed? If so, can we create a plan for doing that?

Reply:
Yes, there's a task in the issue to do that.